### ISSUE TITLE
Release Build Toolchain

### ISSUE THREAD

**Issue by spwilson2** [jdetter/Chronos#13]
- Needed for continuous integration support. #14

**Comment by jdetter** [comment]
Would hosting them publicly on chronos.systems be good enough?

**Comment by spwilson2** [comment]
I would guess that [travis-ci](https://docs.travis-ci.com/) should at least support a git pull or git submodule for building so yeah that should be fine.

**Comment by spwilson2** [comment]
The biggest thing we'd need to work out is that the toolchain build is often somewhat buggy, so we could either host the binaries and headers/libraries on chronos.systems, or we need to fix up the toolchain build a little bit.

**Comment by jdetter** [comment]
We could probably just host the binaries if we can get that figured out

**Comment by spwilson2** [comment]
:+1:  I wouldn't mind taking a look at the toolchain build at some point either, since bringing people on can be a bit of work if the toolchain doesn't build.  I'll wait till the build system is ported to SCons before looking at that though.

**Comment by jdetter** [comment]
If you want to try to get the toolchain ported to SCons that would be cool.

**Comment by spwilson2** [comment]
Do we use anything other than bash for building the toolchain? As far as I remember it was mostly just bash scripts calling make for the build tools and installing prerequisite packages.

**Comment by jdetter** [comment]
It's just bash

**Comment by spwilson2** [comment]
Yeah, I'd say bash is fine then. Will likely just need to touch it up a bit.
